templates/login.html

```diff
@@ -17,13 +17,13 @@
 {% block main %}
 <div class="container" id="home" data-authenticator-login-url="{{authenticator_login_url}}">
   <div class="text-center">
-    <p class="hub-login-text">The 2i2c JupyterHub for {{ hub['org_name'] }}</p><br />
+    <p class="hub-login-text">The 2i2c JupyterHub for <a href="{{ hub['org_url'] }}">{{ hub['org_name'] }}</a></p><br />
     <a href="{{ hub['org_url'] }}">
       <img src="{{ hub['org_logo'] }}" alt="{{ hub['org_name'] }} logo" class='hub-logo'
         title='hub logo' />
     </a>
   <div id="operated-by">
-    Operated by <a href="https://2i2c.org/">2i2c.org</a> as part of the <a href="https://2i2c.org/pilot">2i2c Hubs for All pilot</a>.
+    Operated by <a href="https://2i2c.org/">2i2c.org</a> and <a href="https://www.cloudbank.org/">CloudBank</a> as part of the <a href="https://2i2c.org/pilot">2i2c Hubs for All pilot</a>.
```

Comment:
Not all hubs are paid for by cloudbank credits, no? This should be a configurable option

Reply:
Yeah true - I guess we could add an `operated_by` and a `credits_by` field and insert them here?